CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByGroupAction.java

```diff
@@ -33,7 +33,9 @@
 import org.openide.util.NbBundle.Messages;
 
 @ActionID(category = "Arrange", id = "au.gov.asd.tac.constellation.plugins.arrangements.group.ArrangeByGroupAction")
-@ActionRegistration(displayName = "#CTL_ArrangeByGroupAction", surviveFocusChange = true)
+@ActionRegistration(displayName = "#CTL_ArrangeByGroupAction",
+        iconBase = "au/gov/asd/tac/constellation/plugins/arrangements/group/resources/arrangeByNodes.png",
```

Comment:
Is this icon supposed to be the same as "Arrange by Node" icon?

Reply:
Looks like that's a mistake, I don't know why I did that but I'll remove it.